crates/bevy_winit/src/lib.rs

```diff
@@ -537,6 +538,7 @@ fn handle_create_window_events(
     }
 }
 
+#[cfg_attr(target_os = "android", allow(unused))]
```

Comment:
Should we just compile this out on android; otherwise, this could be used on `android`, but I don't think there's an `unused_allow_unused` lint which would catch that.

Reply:
Maybe I missed  something, but I think we can remove `handle_initial_window_events` at all.